spec/helpers/application_helper_spec.rb

```diff
@@ -19,4 +19,32 @@
       expect(helper.beta_user?).to be false
     end
   end
+
+  describe "#has_unread_notifications?" do
+    let(:notification) { create(:notification, :jam) }
```

Comment:
if we had a default, there would be no need to pass `:jam`, which is not relevant for these specs